torchrl/envs/utils.py

```diff
@@ -181,28 +188,25 @@ def step_mdp(
             next_tensordict.update(out)
             return next_tensordict
         return out
+
     out = tensordict.get("next").clone(False)
-    excluded = set()
+    excluded = {action_key}
     if exclude_done:
         excluded.add(done_key)
     if exclude_reward:
         excluded.add(reward_key)
     if len(excluded):
```

Comment:
that will never happen anymore, so we'll always be calling exclude, which is expensive. Let's avoid that

Reply:
ok